src/App.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from "react";
+import Counter from "./components/Counter/Counter";
```

Comment:
It's common practice to create `index.ts` file in a component's directory and re-export it from there. This way, you can get shorter import paths, e.g. `./components/Counter`